StripeFinancialConnections/StripeFinancialConnections/Source/Helpers/UIColor+Extensions.swift

```diff
@@ -163,14 +163,18 @@ extension UIColor {
         return UIColor(red: 34 / 255.0, green: 132 / 255.0, blue: 3 / 255.0, alpha: 1)  // #228403
     }
 
-    static var linkBrand200: UIColor {
+    static var legacyLinkBrand200: UIColor {
```

Comment:
good rename; I guess these will eventually get deleted once all the colors are replaced? if so, also good to iterate this way instead of changing all the things